app/views/directives/edit-environment-variables.html

```diff
@@ -46,7 +46,7 @@ <h4 class="section-label">
       entries="container.envFrom"
       selector-placeholder="Secret/Config Map"
       env-from-selector-options="$ctrl.valueFromObjects"
-      add-row-link="Add ALL Values from a Resource"
+      add-row-link="Add ALL Variables from Secret or Config Map"
```

Comment:
I think "Add ALL Values..." is more accurate since secrets and config maps don't have "variables."

@beanh66 Are you OK with that change?

Reply:
@spadgett That works for me! So it would be "Add ALL Values from Secret or Config Map" correct?

Reply:
> So it would be "Add ALL Values from Secret or Config Map" correct?

@beanh66 Yup! That's what I was thinking